src/posts/tracker/basics.ts

```diff
@@ -65,7 +65,7 @@ This table provides a fuller explanation for each of the Built-in Person Risk pr
   <tr>
    <td>Avg local resident who works from home
    </td>
-   <td>The known person one may encounter peripherally, like a co-worker who does not interact with the public and can socially distance in an office. There are more interactions than someone who solely works at home, but not as many as someone who works with the public.
+   <td>A known person who has no in-person interactions with the public or other coworkers as part of their job. Treated the same as an average local resident outside of work.
```

Comment:
Why do we include this category? Can we drop it?

"Treated the same as an average local resident outside of work." - I don't understand what this is trying to say.

Reply:
Dropping it sounds cleaner.

If not dropping it, "Has the same errand and socialization patterns as an average local resident." might capture it?

Reply:
It's in the risk tracker built in profile list. It would need to be dropped from there before being dropped from the documentation.